### ISSUE TITLE
Show layers as a strip of ny

### ISSUE THREAD

**Issue by ASKemp** [Dygmalab/Bazecor#93]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by GazHank** [comment]
This sounds interesting, would this perhaps work a bit like tabs?  We could show tabs at the top (or bottom) for the configured layers which you can easily click between, and you could also then easily add remove or reorder the layers at will?

**Comment by ASKemp** [comment]
Yep, just makes it quicker and easier to move between layers than a drop down.

**Comment by alexpargon** [comment]
This seems pretty useful, I'll tag @thiagohernandez so he can attempt to implement it, my take on it would be to do something similar to the typical interface of a carousel, with the layers as numbers or dots at the bottom of the layout.

![image](https://github.com/Dygmalab/Bazecor/assets/14853921/089d2cf5-1394-44a0-8447-d2ca6e840332)
